src/services/casesService/index.ts

```diff
@@ -0,0 +1,90 @@
+import { type Case, type Gender, type Keyword, type Status } from "./model";
+
+const getCaseByID = async (id: number): Promise<Case> => {
+  const res = await fetch(`/api/cases?id=${id}`, {
+    headers: {
+      Accept: "application/json",
+      "Content-Type": "application/json",
+    },
+    method: "GET",
+  });
+
+  const json = await res.json();
+
+  // eslint-disable-next-line prefer-destructuring
+  const caseByID = json[0];
+
+  return caseByID as Case;
```

Comment:
Note that this method can return `undefined` if the specified case is not found (e.g. `id = 0`), since the API result will be empty. Calling functions might need to handle that later. 😅